app/models/concerns/proposal_delegate.rb

```diff
@@ -48,4 +48,8 @@ def self.client
   def client
     self.class.client
   end
+
+  def slug_matches?(user)
```

Comment:
should we write a little unit test for this so that if it changes we have a test failure?